paddle/fluid/platform/mkldnn_reuse.h

```diff
@@ -39,7 +39,8 @@ class MKLDNNHandler {
     std::stringstream ss;
     ss << tid;
     key_ = key_common_ + "-t:" + ss.str();
-    if (platform::get_cur_thread_id() == -1) {
+    if (platform::get_cur_mkldnn_session_id() ==
```

Comment:
Do you need `kMKLDNNSessionID_User`?
`if (platform::get_cur_mkldnn_session_id() !=platform::kMKLDNNSessionID_Default)`?

Reply:
this was to support old api setmkldnnthreadid, now it is deprecated, yes, need to change to your recommend.